client/components/users/UserRankings.jsx

```diff
@@ -1,7 +1,26 @@
+import React, { useState, useEffect } from "react";
 import { Grid, Card, CardContent, CardHeader } from "@material-ui/core";
-import React from "react";
+import { getOverallRankings } from "../../../api_master";
+
+function UserRankings({ loggedInUser }) {
+  const [rankings, setRankings] = useState({});
+
+  var refreshRankings = () => {
+    if (loggedInUser.id) {
+      getOverallRankings(loggedInUser.id).then((res) => {
+        setRankings(res.data);
```

Comment:
Not a big deal, but in past code reviews with Kim and Tom, they said it is better to give this res object a more explicit name, in this case, I'm guessing something like rankings.

Reply:
Addressed